dynamixels: ['head_pan_joint', 'head_tilt_joint']

joints: [right_arm_tilt_joint, right_arm_lift_joint, right_arm_rotate_joint, right_arm_elbow_joint, right_arm_wrist_yaw_joint, right_arm_wrist_tilt_joint, right_arm_gripper_joint, head_pan_joint, head_tilt_mod_joint, left_arm_tilt_joint, left_arm_lift_joint, left_arm_rotate_joint, left_arm_elbow_joint, left_arm_wrist_yaw_joint, left_arm_wrist_tilt_joint, left_arm_gripper_joint, pan_joint]

right_joints: [right_arm_tilt_joint, right_arm_lift_joint, right_arm_rotate_joint, right_arm_elbow_joint, right_arm_wrist_yaw_joint, right_arm_wrist_tilt_joint, pan_joint]

left_joints: [left_arm_elbow_joint, left_arm_lift_joint,left_arm_rotate_joint, left_arm_tilt_joint, left_arm_wrist_tilt_joint, left_arm_wrist_yaw_joint, pan_joint]

head_tilt_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_tilt_joint
    joint_speed: 0.5
    motor:
        id: 10
        init: 512
        min: 0
        max: 1023

head_pan_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_pan_joint
    joint_speed: 0.5
    motor:
        id: 4
        init: 512
        min: 900
        max: 50

right_arm_wrist_yaw_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_arm_wrist_yaw_joint
    joint_speed: 0.5
    motor:
        id: 6
        init: 512
        min: 0
        max: 1023

right_arm_wrist_tilt_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_arm_wrist_tilt_joint
    joint_speed: 0.5
    motor:
        id: 5
        init: 512
        min: 1023
        max: 0

right_arm_gripper_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_arm_gripper_joint
    joint_speed: 0.5
    motor:
        id: 2
        init: 512
        min: 90
        max: 900


left_arm_wrist_tilt_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_arm_wrist_tilt_joint
    joint_speed: 0.5
    motor:
        id: 9
        init: 512
        min: 50
        max: 900

left_arm_wrist_yaw_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_arm_wrist_yaw_joint
    joint_speed: 0.5
    motor:
        id: 7
        init: 512
        min: 1023
        max: 0

left_arm_gripper_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_arm_gripper_joint
    joint_speed: 0.5
    motor:
        id: 8
        init: 512
        min: 900
        max: 90



